### PR TITLE
Hide Actions.

### PR DESCRIPTION
**Pull request by jwineman**
Hide Mint/Burn/Claim etc if on goerli-test.

Screen Shot 2020-10-09 at 3 09 31 PM

### CODE REVIEW

File: src/components/Banner/L2Banner.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { fontFamilies } from 'styles/themes';
 interface L2BannerProps {
 	setCurrentPage: Function;
 }
+
```
Comment (clementbalestrat) [inline src/components/Banner/L2Banner.tsx:12]: @jwineman can you remove that?

File: src/screens/MintrTabs/Home/index.js [hunk 2]
```diff
@@ -1 +1 @@
-export { default } from './Home';
+export { default } from './Home.tsx';
```
Comment (clementbalestrat) [inline src/screens/MintrTabs/Home/index.js:1]: you probably don't need that

File: src/screens/MintrPanel/MintrPanel.js [hunk 3]
```diff
@@ -39,6 +47,11 @@ const MainContainer = ({ currentTab, modalState: { modalType, modalProps }, setC
 							key={tab}
 							isSelected={tab === currentTab}
 							onClick={() => setCurrentTab({ tab })}
+							disabled={
```
Comment (clementbalestrat) [inline src/screens/MintrPanel/MintrPanel.js:50]: Logic is good but a bit dirty. Can you create an object like `RESTRICTED_TABS`

then you'll test if `RESTRICTED_TABS[networkID][tab]` exists in the `disabled` prop

**Review by clementbalestrat — CHANGES_REQUESTED** [review]

File: src/screens/MintrTabs/Home/index.js (hunk 2 above)
Comment (jwineman) [reply src/screens/MintrTabs/Home/index.js:1]:
```
export { default } from './Home';
```
gives me
```
Failed to compile
./src/screens/MintrTabs/Home/Home.js
Error: ENOENT: no such file or directory, open '/Users/johnwineman/synthetix/synthetix-mintr/src/screens/MintrTabs/Home/Home.js'
```

**Review by clementbalestrat — APPROVED** [review]

**Pull request merged**